io.github.Fndroid.clash_for_windows.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: io.github.Fndroid.clash_for_windows
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+base: org.electronjs.Electron2.BaseApp
+base-version: 21.08
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: cfw.sh
+
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=x11
+- --device=dri
+- --filesystem=xdg-config/clash:create
```

Comment:
Are you sure this is needed? Applications usually respect `XDG_CONFIG_HOME`.

Reply:
> Are you sure this is needed? Applications usually respect `XDG_CONFIG_HOME`.

Yes, I'm sure. ~/.config/clash appears to be hardcoded, and not giving the app access to this directory will cause the app's built-in configuration file editor to fail.

Reply:
In that case, it would be preferred to use `--persist=config/clash` here.

Reply:
> In that case, it would be preferred to use `--persist=config/clash` here.

Is it `--persist=.config/clash`, please? I've read the documentation, but after testing, either `--persist=.config/clash` or `--persist=config/clash` will cause the application's built-in configuration editor to not work and the program will not be able to access the configuration files in `~/.config/clash`. I know it's a bit hard to believe, but I've tested it many times and it does.

Reply:
Let's leave it as is, then.